dockerfiles/executor_image/README.md

```diff
@@ -28,7 +28,7 @@ To build a new version of podman from the fork:
 
 ```shell
 # Clone the fork and ensure you have the master branch checked out
-git clone https://github.com/buildbuddy-io/podman-static
+git clone git@github.com:buildbuddy-io/podman-static.git
```

Comment:
this change is fine, but fwiw I use the `gh` CLI which installs a credential helper that makes `https://` work properly

Reply:
ah ok, either way I don't think this matters too much - this PR LGTM :)